dhis-2/dhis-api/src/main/java/org/hisp/dhis/dashboard/DashboardItem.java

```diff
@@ -528,4 +540,96 @@ public void setWidth( Integer width )
     {
         this.width = width;
     }
+
+    /******************************
+     * Deprecated methods required to keep ReportTable and Chart backward compatible
+     ******************************/
+
+    private Visualization convertToVisualization( final Chart chart )
```

Comment:
Instead of converting "real-time", should we instead convert dashboard items once with Flyway?